### PR TITLE
fix linting

### PR DESCRIPTION
**Pull request by mgaonach**

### CODE REVIEW

**Comment by mgaonach** [comment]
@stufisher had to change some variable names in your code and in route parameters to respect python conventions. Is that okay?

**Comment by stufisher** [comment]
Unfortunately here is one of the cases where we want to ignore python standards. We want the function parameters to match the database column names and so cant use snake cased parameters. i.e. we want `blSampleId` rather than `bl_sample_id`. I think we should disable the relevant flake8 rule. I'll dig out the relevant number

File: pyispyb/app/globals.py [hunk 1]
```diff
@@ -1,45 +1,3 @@
-"""
-https://gist.github.com/ddanier/ead419826ac6c3d75c96f9d89bea9bd0
-This allows to use global variables inside the FastAPI application
-using async mode.
```
Comment (stufisher) [inline pyispyb/app/globals.py:4]: I like to leave credit to other peoples code

**Comment by mgaonach** [comment]
> 

Why do we need to exactly match the database name? I think it is pretty obvious that `bl_sample_id` is the same thing as database `blSampleId`.
If we disable the linting, then we will not get any warning when we want the naming convention to be respected, i.e. anywhere else in the code, which in a few month of development will end-up in having no convention at all...

**Comment by stufisher** [comment]
Because it provides consistency for developers. They only have to remember one definition of blSampleId, not bl_sample_id, blsampleid. We did this before in synchweb having aliased versions of the column names and it is an absolute nightmare for maintenance, no one knows how anything is defined in the project. If you stick to the column names it keeps it simple for everyone. Using snake casing is not particularly important, its mostly that naming is consistent.

cc @JPHall-DLS @rjgildea @KarlLevik

**Comment by stufisher** [comment]
Because FastAPI converts the parameters to query and body arguments, it also means the api will then use `bl_sample_id` and not `blSampleId` and certainly on the API side we want it to reflect the database column style

**Comment by stufisher** [comment]
PEP8 allows mixedCase https://peps.python.org/pep-0008 in specific cases, and i think ours is valid
```
mixedCase is allowed only in contexts where that’s already the prevailing style (e.g. threading.py), to retain backwards compatibility.
```

**Comment by mgaonach** [comment]
They also state that internal consistency should be preferred over the standards - how about using camel case everywhere then?
```
New modules and packages (including third party frameworks) should be written to these standards, but where an existing library has a different style, internal consistency is preferred.
```

**Comment by stufisher** [comment]
Yes agreed, i will see if there is a way to enforce mixed/camelCasing with the linter

**Comment by JPHall-DLS** [comment]
I think the developer experience is better with mixedCase consistent across the database, API, and front end.
My preference would also be for the API to reflect database column names. The exception should be UPPERCASE columns in the database that we agree should really be mixedCase. For example, BLSample.POSITIONID, DataCollection.BLSAMPLEID, DataCollection.SESSIONID, etc. Perhaps we could override these in the model?

**Comment by stufisher** [comment]
The uppercase columns are deprecated and marked for removal by @KarlLevik

**Pull request closed without merging**